docs/docs/Configuration/configuration-objects.md

```diff
@@ -1,34 +1,57 @@
 ---
-title: Data & Message
-sidebar_position: 2
-slug: /guides-data-message
+title: Langflow Objects
+sidebar_position: 1
+slug: /configuration-objects
 ---
 
+## Message object
```

Comment:
What other types of objects are in Langflow that users might want to know about?